### ISSUE TITLE
flutter build apk -v

### ISSUE THREAD

**Issue by Tonigt** [flutter/flutter#104413]
<details>
<summary>Logs</summary>

```
[ +287 ms] Downloading https://services.gradle.org/distributions/gradle-6.7-all.zip
[+27130 ms] ...............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[   +7 ms] Unzipping /Users/tonight/.gradle/wrapper/dists/gradle-6.7-all/cuy9mc7upwgwgeb72wkcrupxe/gradle-6.7-all.zip to /Users/tonight/.gradle/wrapper/dists/gradle-6.7-all/cuy9mc7upwgwgeb72wkcrupxe
[+3977 ms] Set executable permissions for: /Users/tonight/.gradle/wrapper/dists/gradle-6.7-all/cuy9mc7upwgwgeb72wkcrupxe/gradle-6.7/bin/gradle
[+2873 ms] Welcome to Gradle 6.7!
[        ] Here are the highlights of this release:
[        ]  - File system watching is ready for production use
[        ]  - Declare the version of Java your build requires
[        ]  - Java 15 support
[        ] For more details see https://docs.gradle.org/6.7/release-notes.html
[  +80 ms] Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details
[+447809 ms] FAILURE: Build failed with an exception.
[   +9 ms] * What went wrong:
[        ] A problem occurred configuring project ':appscheme'.
[        ] > Could not resolve all artifacts for configuration ':appscheme:classpath'.
[        ]    > Could not resolve org.ow2.asm:asm-util:7.0.
[        ]      Required by:
[        ]          project :appscheme > com.android.tools.build:gradle:4.1.0
[        ]          project :appscheme > com.android.tools.build:gradle:4.1.0 > com.android.tools.build:builder:4.1.0
[        ]       > Could not resolve org.ow2.asm:asm-util:7.0.
[        ]          > Could not get resource 'https://jcenter.bintray.com/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom'.
[        ]             > Could not HEAD 'https://jcenter.bintray.com/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom'.
[        ]                > Remote host terminated the handshake
[        ]    > Could not resolve org.ow2.asm:asm-util:7.0.
[        ]      Required by:
[        ]          project :appscheme > com.android.tools.build:gradle:4.1.0 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
[        ]       > Could not resolve org.ow2.asm:asm-util:7.0.
[        ]          > Could not get resource 'https://jcenter.bintray.com/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom'.
[        ]             > Could not HEAD 'https://jcenter.bintray.com/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom'.
[        ]                > Remote host terminated the handshake
[        ] > Failed to notify project evaluation listener.
[        ]    > Could not get unknown property 'android' for project ':appscheme' of type org.gradle.api.Project.
[        ]    > Could not get unknown property 'android' for project ':appscheme' of type org.gradle.api.Project.
[        ] * Try:
[        ] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
[        ] * Get more help at https://help.gradle.org
[        ] BUILD FAILED in 8m 2s
[        ] Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
[        ] Use '--warning-mode all' to show the individual deprecation warnings.
[        ] See https://docs.gradle.org/6.7/userguide/command_line_interface.html#sec:command_line_warnings
[ +655 ms] Running Gradle task 'assembleRelease'... (completed in 483.1s)
[   +4 ms] [!] Gradle threw an error while downloading artifacts from the network. Retrying to download...
[+2249 ms] Running Gradle task 'assembleRelease'...
[   +3 ms] Using gradle from /Users/tonight/Desktop/HJNlIVE/hjnzb/android/gradlew.
[  +24 ms] executing: [/Users/tonight/Desktop/HJNlIVE/hjnzb/android/] /Users/tonight/Desktop/HJNlIVE/hjnzb/android/gradlew -Pverbose=true -Ptarget-platform=android-arm,android-arm64,android-x64 -Ptarget=lib/main.dart -Pbase-application-name=android.app.Application -Pdart-obfuscation=false -Ptrack-widget-creation=true -Ptree-shake-icons=true assembleRelease
[ +304 ms] Downloading https://services.gradle.org/distributions/gradle-6.7-all.zip

```
</details>

<img width="1191" alt="WeChat60ec3f02d8395f2c0d3d2296c5713eda" src="https://user-images.githubusercontent.com/22694931/169830307-10f0d858-991a-4659-80bf-ba54a8a306f0.png">
--------------------

so can you tell me why it's just a .pom can't access but delete grade again 
i can't understand 

in  android project its build complete, why flutter build apk delete it,

**Comment by Tonigt** [comment]
[✓] Flutter (Channel stable, 2.10.5, on macOS 12.3.1 21E258 darwin-x64, locale zh-Hans-CN)
    • Flutter version 2.10.5 at /Users/tonight/fvm/versions/2.10.5
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5464c5bac7 (5 weeks ago), 2022-04-18 09:55:37 -0700
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0-rc4)
    • Android SDK at /Users/tonight/Library/Android/sdk
    • Platform android-31, build-tools 30.0.0-rc4
    • ANDROID_HOME = /Users/tonight/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.66.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.40.0

[✓] Connected device (2 available)
    • MRX AL09 (mobile) • KBX6R19C04002712 • android-arm64 • Android 10 (API 29)
    • macOS (desktop)   • macos            • darwin-x64    • macOS 12.3.1 21E258 darwin-x64

[!] HTTP Host Availability
    ✗ HTTP host https://maven.google.com/ is not reachable. Reason: An error occurred while checking the HTTP host: Operation timed out

! Doctor found issues in 1 category.

**Comment by huycozy** [comment]
Hi @Tonigt,
It seems you are using [appscheme](https://pub.dev/packages/appscheme) package/plugin, right?
There is a similar issue here: #83851 and it's solved by the solution in the [comment](https://github.com/flutter/flutter/issues/53977#issuecomment-609699286).
Please try the solution and see if it works for you.

**Comment by Tonigt** [comment]
just relation to network, spend long time to update dependence, flutter build apk delete it and redownload what i don't known why。

**Comment by huycozy** [comment]
@Tonigt: 
> [!] HTTP Host Availability
    ✗ HTTP host https://maven.google.com/ is not reachable. Reason: An error occurred while checking the HTTP host: Operation timed out

From your `flutter doctor -v`, please ensure you have a working internet connection.
There is a checklist you can use for this:

- [ ] Is the network stable and fast?
- [ ] Do you use any proxy server?
- [ ] Are you using https://flutter.dev/community/china#configuring-flutter-to-use-a-mirror-site?

Some solved solutions that may help you: #101095, #98579, #60691

If these solutions are not working for you, please provide a [completed and minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) that we can directly copy paste and run to verify this issue.
